src/west/manifest.py

```diff
@@ -35,6 +35,10 @@
 #: A fully qualified reference to `MANIFEST_REV_BRANCH`.
 QUAL_MANIFEST_REV_BRANCH = 'refs/heads/' + MANIFEST_REV_BRANCH
 
+#: The west refs space used for current manifest revision.
+#: West ref space is temporary space for fetching of SHAs.
+WEST_REF_SPACE = 'refs/west/'
```

Comment:
Should we call this something like QUAL_REFS_WEST for consistency with QUAL_MANIFEST_REV_BRANCH?

Reply:
Consistency is king :+1: